src/main/java/org/jpeek/XslReport.java

```diff
@@ -135,16 +140,18 @@
      * @param xml Skeleton
      * @param name Name of the metric
      * @param args Params for XSL
+     * @param calc Calculus
      * @param mean Mean
      * @param sigma Sigma
      * @checkstyle ParameterNumberCheck (10 lines)
      */
     XslReport(final XML xml, final String name,
-        final Map<String, Object> args,
+        final Map<String, Object> args, final Calculus calc,
```

Comment:
@HDouss This constructor IMHO contains too many parameter, it 5 and now it is 6.
I personally think the idea would be 3, max 4. 
What do you think about the possibility of refactoring that as well ??? Or leave a todo???

Reply:
@fanifieiev Done!